### PR TITLE
E 04372 trdm

### PR DESCRIPTION
**Pull request by cameroncaci**
This pull request is the feature branch of TRDM. All previous code has been peer reviewed and approved. Remember, this originally came from the previous caci milmove repository.
[LastTableUpdate PR](https://github.com/transcom/mymove/pull/11543)
[GetTable PR](https://github.com/transcom/mymove/pull/11551)
[TRDM Secret Refactor](https://github.com/CACI-MIlMOVE/mymove/pull/24)
[Soap Security Header](https://github.com/CACI-MIlMOVE/mymove/pull/13)
[Golang CI Lint](https://github.com/CACI-MIlMOVE/mymove/pull/4)
[Loa Parser](https://github.com/CACI-MIlMOVE/mymove/pull/2)
[Tac Parser](https://github.com/CACI-MIlMOVE/mymove/pull/1)

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -2,3 +2,5 @@
 
 - [MilMove Docs](https://transcom.github.io/mymove-docs) contains the primary set of documents for the project
 - [Architectural Decision Records](https://transcom.github.io/mymove-docs/docs/adrs/) or ADRs are the list of technical decisions for the project.
+
+Tests-Tevin Adams
```
Comment (deandreJones) [inline docs/README.md:6]: is this needed

Comment (cameroncaci) [reply docs/README.md:6]: I'm not sure, that was likely auto generated. Myself, Jackson, and Tevin are all code contributors. I only joined TRDM when the security header popped up (Besides the parser packages).

@TevinAdams this one's a question for you

Comment (danieljordan-caci) [reply docs/README.md:6]: Will approve once this gets addressed

Comment (TevinAdams) [reply docs/README.md:6]: No that's not needed. It was a test push to a read.me when we first started.  I removed it so not sure how it got to this branch.

**Review by traskowskycaci — APPROVED** [review]

File: docs/README.md (hunk 1 above)
Comment (cameroncaci) [reply docs/README.md]: Removed line

**Review by deandreJones — APPROVED** [review]

**Pull request merged**